client/components/SideBar/Sidebar.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import './SideBar.scss'
+
+const SideBar = () => {
+  const categories = [
+    {
+      title: 'pizza',
+      path: '/',
+    },
+
+    {
+      title: 'drinks',
+      path: '/',
+    },
+  ]
+
+  return (
+    <aside className="sidebar">
+      <span>categories:</span>
+      <nav>
+        <ul className="sidebar__nav">
+          {categories.map((el, idx) => (
+            <li
+              className={`sidebar__nav-link ${
+                el.title === 'pizza' ? 'active' : ''
+              }`}
+              key={idx.toString()}
```

Comment:
@AStashevskaya Lets avoid using `indexes` as `keys` if we can any other `unique` alternative, like `el.path`, and also try to use more declarative variable names, for example if you have `{categories.map()}`, then you can use `category` as name of variable